### ISSUE TITLE
Pension List+Loop: Interactive not selectable on page load using MacOS Voice Control + names or flags

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#105389]
## Brief description
As part of our list and loop enhancements, we submitted an [ADE ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/104338) for a11y review. 

## Problem - Conditional alerts are invisible to screenreader users

**Page**: Seems to be an issue across the entire form?
**Device**: MacOS
**Expected severity level**: Honestly not sure, since you can still use the number grid to select items.
**Expected experience standard**: User experiences predictable interactions with components and patterns
**Description**: 
I encountered a strange issue using Safari + MacOS Voice Control (speech recognition). Essentially, when navigating around using Voice Control, if I go to the next page of the form (press “Continue”), I’m unable to use names or flags/numbers to select form elements on load. I have to click into the screen, scroll, or do another mouse action to “turn on” the flags. 
[Example 1](https://us02web.zoom.us/clips/share/_AFvZp7YSiCV6VV5lMskPQ)
[Example 2](https://us02web.zoom.us/clips/share/7XZ05-0DQlSBiBvy4ZB2NQ)
This was not an issue using Chrome + Voice Control, or using the Windows equivalent. 
I also couldn’t duplicate this issue using a couple of other VA forms I tried, so it might be unique to this form.
This is an issue because people using MacOS Voice Control are likely using it because they have issues with their hands, or can’t use their hands. If you have to use your mouse too, it’s at best uncomfortable, and at worst impossible.

**How to recreate**: 

1. Turn on Voice Control (Settings > Accessibility > Voice Control)
2. In the Voice Control settings, make sure flags/numbers are turned on (“Overlay: Item Numbers”)
3. Start filling out the form, then use Voice Control to select “Continue” - do flags appear on the next page? Can you select form items using their names?
4. Click on the page - do the flags pop in?

**Recommendation**: Honestly not sure, I haven’t encountered this yet. I spoke with Ryan Smith who said “My guess is something to do with how the content gets loaded - maybe the elements aren’t in the DOM when the page is rendered or get added in an atypical way, so then Voice Control isn’t picking them up immediately. But I think some investigation might be needed.” We were hoping that a dev from your team could play with this - if not, we’ll report to the VFF team.
